contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -10,12 +10,12 @@ import {ITrueFiPool2} from "./interface/ITrueFiPool2.sol";
 import {ITrueCreditAgency} from "./interface/ITrueCreditAgency.sol";
 import {ITrueFiCreditOracle} from "./interface/ITrueFiCreditOracle.sol";
 import {ITimeAveragedBaseRateOracle} from "./interface/ITimeAveragedBaseRateOracle.sol";
+import {IBorrowingMutex} from "./interface/IBorrowingMutex.sol";
 
 interface ITrueFiPool2WithDecimals is ITrueFiPool2 {
     function decimals() external view returns (uint8);
 }
 
-// prettier-ignore
```

Comment:
I would also like to remove `prettier-ignore`, but it looks like prettier deletes all our comments for this contract.

Personally, I'm in favor of deleting these comments since I prefer code to be self-documenting. (Documentation should be used sparingly to answer _why_ code is designed a particular way instead of duplicating _what_ it does, in a way that might go stale.)

However, keeping these comments is a hard requirement from above, so we can't just remove it. Until the comment deletion issue is fixed, we have to keep `prettier-ignore` here and keep all the comments.